src/components/Slideshow.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+
+import request from 'superagent';
+
+import './Slideshow.css';
+
+const url = 'https://cdn.contentful.com/';
+const spaceId = 'v441z4n7clri';
+const environmentId = 'master';
+const accessToken = '2uQkP24LThdDkW6JWmvVIA3ys7nqBwX6df_fzamh9JU';
+
+let slideshowCounter = 0;
+
+class Slideshow extends Component {
+  constructor(props) {
+    super(props);
+    this.state = 
+      { 
+        currentImage: {},
+      };
+  }
+
+  componentDidMount() {
+    request
+      .get(`${url}spaces/${spaceId}/environments/${environmentId}/entries?access_token=${accessToken}`)
+      .query({'content_type': 'images'})
+      .set('Authorization', `Bearer: ${accessToken}`)
+      .then(({ body: { items } }) => {
+        this.setState({ images: items });
+        this.setState({ currentImage: items[0].fields.gallery[0].url});
+      });
```

Comment:
this.setState({ images: gallery });
this.setState({ currentImage: gallery[0].url })